src/helpers/issue.ts

```diff
@@ -133,6 +133,31 @@ export const getAllIssueComments = async (issue_number: number): Promise<Comment
   return result;
 };
 
+export const issueAutopayCheck = async (issue_number: number) => {
+  let comments = await getAllIssueComments(issue_number);
+  let lastAutopayStatus = null;
+
+  const regex = /Autopay is (disabled|enabled)/;
+
+  for (const item of comments) {
+    const { body, user } = item;
+    if (user?.login === "ubiquibot[bot]" && user?.type === "Bot") {
```

Comment:
if (user?.type === "Bot") {
```suggestion
    if (user?.login === "ubiquibot[bot]" && user?.type === "Bot") {
```

Reply:
Your suggestion doesn't seem right. It means that it will run this check on any bot comment, right?

Reply:
> Your suggestion doesn't seem right. It means that it will run this check on any bot comment, right?

I added that so it only comes fromubiquibot, i feared someone could exploit by commenting as a bot.
 
But bots are github apps.. and there is no they are exploiting any of the github apps installed just to comment autopay

Reply:
>  and there is no they are exploiting any of the github apps installed just to comment autopay

I don't understand your statement.

Reply:
> > and there is no they are exploiting any of the github apps installed just to comment autopay
> 
> I don't understand your statement.

It means to comment as a bot, you need a github app installed on the repo and that is very unlikely..

Reply:
Installing any GitHub App on the repo is very easy to do I don't understand what you're talking about.

Reply:
But can only be installed by admin, so that neutralises the risks..

Reply:
@0xcodercrane I don't understand this issue what do you think?